_pages/order-lib.md

```diff
@@ -4,19 +4,46 @@ title: Order Lib Release Notes
 
 Spartacus 4.2 introduces the Order Library. This is the latest iteration of the code splitting initiative. The main goal of code splitting is to reduce the initial download size of a Spartacus enabled app by leveraging lazy loading of feature libraries.
 
-# Backwards compatibility for existing apps by default
+# Existing Apps upgrading to 4.2: keep backwards compatibility
 
-By default, if you upgrade to 4.2 from 4.x, the order library will not be used. Instead, your app will continue to use the old/current order related code. The non-lib order code is still present in Spartacus and will only be removed in 5.0. This allows Spartacus 4.2 to be backwards compatible with previous 4.x versions.
+The the new Order Library is not be backwards compatible. Therefore, by default, if you upgrade to 4.2 from 4.x, the order library will NOT be used. Instead, your app will continue to use the old/current order related code to preserve the backwards coompatibility of the upgrade. The non-lib order code is still present in Spartacus. This allows Spartacus 4.2 to be backwards compatible with previous 4.x versions.
 
 # Order lib used in new Spartcus Apps.
 
-In Spartacus 4.2, if you generate a new Spartacuus app with the schematics, the app will use the new Order Library.
+In Spartacus 4.2, if you generate a new Spartacus app with the schematics, the app will use the new Order Library.
+
+# Order lib used in new Spartcus Apps.
+
+The the new Order Library is not be backwards compatible. Some changes to the code were required to make the order related features work within the order library. If you have custom code iin your app, theere is a chance that this code will require changees for it to work with the oorder library. If you still want to explore the possibility of using the order lib in an existing app upgraded to 4.2, you can switch to use the cart lib by changing the imports. ( in a standard app structure, these imports are in )
+
+You first need to remove the imports of the order code that was moved.
```

Comment:
customers may not just import modules, there are also other stuff (models...). I don't know whether we need give these detailed stuff.